# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/xmms-crystality/prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur crystality-plugin-0.92.orig/Makefile crystality-plugin-0.92/Makefile
--- crystality-plugin-0.92.orig/Makefile	2001-08-13 01:41:14.000000000 +0200
+++ crystality-plugin-0.92/Makefile	2004-10-23 19:10:26.000000000 +0200
@@ -35,9 +35,7 @@
 	  echo "No Crystality binaries found, do \"make\" before\!";\
 	  exit 1;\
 	fi ;\
-	if [ -d /usr/local/bin ] ; then \
-	  cp $(STDIO_PLUGIN) /usr/local/bin ;\
-	elif [ -d /usr/bin ] ; then \
+	if [ -d /usr/bin ] ; then \
 	  cp $(STDIO_PLUGIN) /usr/bin; \
         else \
 	  echo -e "\n\n**********************************************************" ;\
diff -Nur crystality-plugin-0.92.orig/README crystality-plugin-0.92/README
--- crystality-plugin-0.92.orig/README	2001-08-13 01:53:35.000000000 +0200
+++ crystality-plugin-0.92/README	2004-10-23 19:10:37.000000000 +0200
@@ -66,7 +66,7 @@
 Distribution contains binary version of XMMS plugin library and stdin/stdout
 plugin executable (Linux i586, glibc 2.1.3).
 You may copy plugin library file (libcrystality.so) into XMMS's Effect
-directory and executable (crystality-stdio) to /usr/local/bin or any location
+directory and executable (crystality-stdio) to /usr/bin or any location
 you prefer. For default locations simply type:
 
 make install
